db/test-data.sql

```diff
@@ -1,2 +1,94 @@
-INSERT INTO `nk2_shows` (`id`, `title`, `venue_id`, `external_store_description`, `time`, `active`, `inactivateTime`, `description`, `customer_id`, `url`) VALUES
-(1, 'Ensi-ilta', 1, '', '2011-04-19 19:00:00', 1, '2011-04-18 12:00:00', 'Parhaat katsomopaikat ovat permannolla.<br><br>Paikalle kannattaa saapua hyvissä ajoin ruuhkan välttämiseksi. Osoite: Aleksanterin teatteri, Helsinki, Bulevardi 23-27 / Albertinkatu 32', 0, '');
+start transaction;
+
+insert into nk2_discount_codes (code, eur, use_max, email, code_group) values
+    ('TEST_123', '14', 1, 'foo@example.com', 'TEST'),
+    ('admin-code', 99, 0, 'pekko@example.com', 'ADMIN');
+
+insert into nk2_discount_groups (show_id, title, eur, admin_only, active) values
+    (null, 'Opiskelija', 4, false, true),
+    (null, 'Ilmaislippu', 999, true, true),
+    (1, 'Erikoislippu', 2, false, true),
+    (1, 'Eläkeläinen', 5, false, false);
+
+insert into nk2_venues (title, ticket_type, description) values
+    ('Aleksanterin teatteri', 'numbered-seats', 'Bulevardi xx, Helsinki\nRatikat y,z'),
+    ('Sigyn-sali', 'generic', 'xxkatu yy, Turku');
+
+insert into nk2_sections_static (venue_id, title, row_name) values
+    (1, 'Permanto, parhaat paikat', 'riiv'),
+    (1, 'Permanto', 'rivi'),
+    (1, '1. parvi', 'rivi'),
+    (1, 'Aitiot, 1.parvi', 'aitio'),
+
+    (2, 'Permanto', 'riiv');
+
+insert into nk2_seats_static (section_static_id, row, number, x_coord, y_coord, bad_seat) values
+    (1, 1, 1, 10, 10, false),
+    (1, 1, 3, 30, 10, false),
+    (2, 1, 2, 20, 10, false),
+    (3, 5, 78, 40, 150, true),
+    (4, 5, 2, 100, 80, false),
+
+    -- generic tickets - seats not numbered
+    -- there might be a better way to do this
+    (5, null, null, null, null, false),
+    (5, null, null, null, null, false),
```

Comment:
Does this mean that there is one line for each seat? I really don't like the idea, we should come up with something better. Do we want to support non-numbered seats with multiple sections? One way would be to add to nk2_sections_static two ints (total_number_of_seats, number_of_inactive_seats) that would be null if the seats are numbered.